oshmem/mca/spml/spml.h

```diff
@@ -314,6 +314,29 @@ typedef int (*mca_spml_base_module_send_fn_t)(void *buf,
                                               int dst,
                                               mca_spml_base_put_mode_t mode);
 
+/**
+ * The routine transfers the data asynchronously from the source PE to all PEs
+ * in the OpenSHMEM job. The routine returns immediately. The source and target
+ * buffers are reusable only after the completion of the routine. After the data
+ * is transferred to the target buffers, the counter object is updated atomically.
+ * The counter object can be read either using atomic operations such as
+ * shmem_atomic_fetch or can use point-to-point synchronization routines such as
+ * shmem_wait_until and shmem_test.
+ *
```

Comment:
@brminich Reading again, the below text might sound better. Main idea is to add entry and exit conditions.
Thoughts ?

> The routine transfers the data asynchronously from the calling PE to all PEs in the OpenSHMEM job. The routine returns immediately.  **Upon entry, the following conditions should be true. The counter value is a valid integer value and target buffers should be ready to receive the data.  Upon completion of the routine, the following conditions are true on the remote PEs. The counter object is incremented and target symmetric data object is updated. On the local PE, the counter object and target symmetric data object state is undefined.** The counter object can be read either using atomic operations such as shmem_atomic_fetch or using point-to-point synchronization routines such as shmem_wait_until and shmem_test. The target and source are reusable only after the completion of the routine.